pkgs/development/python-modules/ics/default.nix

```diff
@@ -9,21 +14,35 @@ buildPythonPackage rec {
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
-    owner = "C4ptainCrunch";
+    owner = "ics.py";
     repo = "ics.py";
```

Comment:
The upstream repo appears to be `github.com/ics-py/ics-py`

Reply:
I guess that it would help if I clean my :eyeglasses: 

Thanks, updated.

Reply:
Hehe, LGTM now :)